utils/ccdatofhir/mapping_datatypes.bal

```diff
@@ -0,0 +1,218 @@
+            streetAddressLineElement = addressElement.children().<v3:streetAddressLine>;
+            cityElement = addressElement.children().<v3:city>;
+            stateElement = addressElement.children().<v3:state>;
+            postalCodeElement = addressElement.children().<v3:postalCode>;
+        } else {
+            streetAddressLineElement = addressElement.children().<streetAddressLine>;
+            cityElement = addressElement.children().<city>;
+            stateElement = addressElement.children().<state>;
+            postalCodeElement = addressElement.children().<postalCode>;
+        }
+
+        if streetAddressLineElement is xml:Element && streetAddressLineElement.length() > 0 {
+            address.line = [streetAddressLineElement.get(0).data()];
+        }
+
+        if cityElement is xml:Element && cityElement.length() > 0 {
+            address.city = cityElement.get(0).data();
+        }
+
+        if stateElement is xml:Element && stateElement.length() > 0 {
+            address.state = stateElement.get(0).data();
+        }
+
+        if postalCodeElement is xml:Element && postalCodeElement.length() > 0 {
+            address.postalCode = postalCodeElement.get(0).data();
+        }
+        return address;
+    }
+    return;
+}
+
+public isolated function mapCcdaTelecomToFhirTelecom(xml<xml:Element> telecomElement) returns r4:ContactPoint|error? {
+    r4:ContactPoint telecom = {};
+    if telecomElement is xml:Element && telecomElement.length() > 0 {
+        string|error telecomUse = telecomElement.use;
+        string|error telecomValue = telecomElement.value;
+
+        if telecomValue is string {
+            telecom.value = telecomValue;
+        }
+
+        if telecomUse is string {
+            match telecomUse {
+                "HP" => {
+                    telecom.use = "home";
+                }
+                "WP" => {
+                    telecom.use = "work";
+                }
+            }
+        }
+    }
+    return telecom;
+}
+
+public isolated function mapCcdaNametoFhirName(xml<xml:Element> nameElement, boolean isNamespaceAvailable = true) returns r4:HumanName|error? {
+    if nameElement is xml:Element && nameElement.length() > 0 {
+        r4:HumanName name = {};
+        xml<xml:Element> familyElement;
+        xml<xml:Element> givenElement;
+        if isNamespaceAvailable {
+            familyElement = nameElement.children().<v3:family>;
+            givenElement = nameElement.children().<v3:given>;
+        } else {
+            familyElement = nameElement.children().<family>;
+            givenElement = nameElement.children().<given>;
+        }
+        if familyElement is xml:Element && familyElement.length() > 0 {
+            name.family = familyElement.get(0).data();
+        }
+
+        if givenElement is xml:Element && givenElement.length() > 0 {
+            name.given = [givenElement.get(0).data()];
+        }
+        return name;
+    }
+    return;
```

Comment:
Not needed, check the other places

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/bd15548be31a7e76b4962973508cdedd6f8fd58a

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/9d1af1abf6324613105390b679346544139bb847